plugins/catalog/src/alpha/apis.tsx

```diff
@@ -48,4 +52,13 @@ export const catalogStarredEntitiesApi = createApiExtension({
   }),
 });
 
-export default [catalogApi, catalogStarredEntitiesApi];
+export const entityPresentationApi = createApiExtension({
+  factory: createApiFactory({
+    api: entityPresentationApiRef,
+    deps: { catalogApiImp: catalogApiRef },
+    factory: ({ catalogApiImp }) =>
+      DefaultEntityPresentationApi.create({ catalogApi: catalogApiImp }),
```

Comment:
```suggestion
    deps: { catalogApi: catalogApiRef },
    factory: ({ catalogApi }) =>
      DefaultEntityPresentationApi.create({ catalogApi }),
```

just for uniformity

Reply:
Yeaaaah, but `catalogApi` is defined in the parent scope.

Reply:
Well that's a bug in our naming convention 😅, making a note of it and this is obv fine to ship as is 👍